### PR TITLE
Piotrp msft/issue1080/project names

### PR DESCRIPTION
**Pull request by piotrpMSFT**

### CODE REVIEW

File: scripts/compile/compile-stage.sh [hunk 1]
```diff
@@ -24,22 +24,22 @@ source "$DIR/../common/_common.sh"
...
+    dotnet-repl \
+    dotnet-repl-csi \
     dotnet-restore \
-    Microsoft.DotNet.Tools.Resgen \
-    Microsoft.DotNet.Tools.Run \
-    Microsoft.DotNet.Tools.Test \
+    dotnet-resgen \
+    dotnet-run \
+    dotnet-test \
+    Microsoft.DotNet.ProjectModel.Server \
```
Comment (davidfowl) [inline scripts/compile/compile-stage.sh]: fix this too

File: src/dotnet-compile-native/appdep/project.json [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
-    "name": "appdepsdk",
```
Comment (davidfowl) [inline src/dotnet-compile-native/appdep/project.json]: You might have to fix anywhere that assumes something about this name.

File: scripts/compile/compile-stage.sh (hunk 1 above)
Comment (piotrpMSFT) [inline scripts/compile/compile-stage.sh]: good thing we're changing naming conventions. The old one was tough to follow!

File: src/dotnet-repl/project.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
-    "name": "dotnet-publish",
```
Comment (livarcocc) [inline src/dotnet-repl/project.json]: Did you move this to the right place? It says src/dotnet-repl/project.json above.

Comment (piotrpMSFT) [inline src/dotnet-repl/project.json]: yes. fixed someone else's copy/paste

File: src/dotnet-build/project.json [hunk 4]
```diff
@@ -8,7 +7,7 @@
         "NETStandard.Library": "1.0.0-rc3-23727",
         "Microsoft.DotNet.ProjectModel": "1.0.0-*",
         "Microsoft.DotNet.Compiler.Common": "1.0.0-*",
-        "Microsoft.DotNet.Tools.Compiler": "1.0.0-*",
+        "dotnet-compile": "1.0.0-*",
```
Comment (anurse) [inline src/dotnet-build/project.json]: Slightly unrelated, but it seems odd that `dotnet-build` needs a dependency on `dotnet-compile`

File: src/dotnet-run/project.json [hunk 5]
```diff
@@ -9,14 +8,14 @@
...
-        "System.CommandLine" : "0.1.0-d111815-3",
+        "System.CommandLine" : "0.1.0-e160119-1",
 
         "Microsoft.DotNet.ProjectModel": "1.0.0-*",
         "Microsoft.DotNet.Cli.Utils": "1.0.0-*"
     },
     "frameworks": {
         "dnxcore50": {
-            "imports": "portable-net451+win8"
+            "imports": "portable-net45+win8+wpa81+aspnetcore50"
```
Comment (anurse) [inline src/dotnet-run/project.json]: Is this required? AFAIK `portable-net45+win8` should be enough. You just have to capture a subset of the items, and you can only choose one subset right now, so `portable-net45+win8` is the broadest set :).

**Comment by anurse** [comment]
Looks good to me, minor comments and obviously we should figure out why the build is failing :). I'll wait on this before merging #1077

**Comment by anurse** [comment]
Looks like some of the `imports` are pointing to `portable-net451+win8` when they should say `portable-net45+win8`. A global find/replace should fix that TBH.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OS X Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OS X Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Ubuntu Release Build please.

**Comment by anurse** [comment]
FYI, MyGet appears to be down, causing restore failures.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Ubuntu Release Build please.

**Comment by anurse** [comment]
@dotnet-bot test Windows_NT Release Build please

**Comment by anurse** [comment]
@dotnet-bot test Ubuntu Debug Build please

**Comment by anurse** [comment]
`build_appdeps` needs to have the folder name updated.

**Comment by piotrpMSFT** [comment]
@anurse yea, updated it. There is also a native runtime issue folks noticed today... not sure the cause, but fixed it.

**Comment by anurse** [comment]
:+1:. @davidfowl and I are still mildly concerned that merging our two PRs is going to be scary, but less scary if you go first, so I'm going to hang on a bit longer and wait it out :)

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX Release Build please.

**Pull request merged**